app/assets/javascripts/admin/addon/components/admin-plugin-config-page.gjs

```diff
@@ -1,19 +1,19 @@
 import Component from "@glimmer/component";
 import { inject as service } from "@ember/service";
+import HorizontalOverflowNav from "discourse/components/horizontal-overflow-nav";
+import NavItem from "discourse/components/nav-item";
 import i18n from "discourse-common/helpers/i18n";
+import I18n from "discourse-i18n";
```

Comment:
🤯 

How did we end up here? Should we have a TODO to consolidate this?

Reply:
Yeah it kinda irks me each time...but one is an ember template helper (e.g. `{{i18n "blah"}}`) and one is a general `I18n` library that can be used in JS. Will open a dev topic about it, maybe there is something that can be done? Like `import { I18n, i18nHelper } from "discourse-i18n"`...or something

Reply:
the helper is just a function that calls `I18n.t()` so you can use just that 🙂